code/modules/projectiles/projectile_base.dm

```diff
@@ -62,6 +62,8 @@
 	var/jitter = 0
 	/// Number of times an object can pass through an object. -1 is infinite
 	var/forcedodge = 0
+	/// Does the projectile increase fire stacks / immolate mobs on hit?
+	var/immolate = 0
```

Comment:
What does a higher value mean as opposed to 0 or 1?

Reply:
this is the number of fire stacks the variable will apply, will make that more clear on  the autodoc

Reply:
Done